modules/automatic-upgrade-failure.adoc

```diff
@@ -1,7 +1,7 @@
 // Module included in the following assemblies:
 //
 // * configuration/configure-automatic-upgrades.adoc
-:_module-type: CONCEPT
+:_mod-docs-content-type:CONCEPT
```

Comment:
You need a space here.
```suggestion
:_mod-docs-content-type: CONCEPT
```

It looks like all of the ones that were ":_content-type:" are correct, but the ones that were ":_module-type:"  all need a space.